test/Sema/issue-46000.swift

```diff
@@ -0,0 +1,19 @@
+// RUN: %target-typecheck-verify-swift
+
+// https://github.com/apple/swift/issues/46000
+
+struct DispatchData {
+  let ptr: UnsafePointer<UInt8>
+  func withUnsafeBytes<Result, ContentType>(body: (UnsafePointer<ContentType>) throws -> Result) rethrows -> Result { try body(ptr as! UnsafePointer<ContentType>) }
```

Comment:
FYI: If a function, like this one, is defined just for its type signature in a semantic analysis test, the body can be left empty: missing returns are diagnosed in SIL.

Reply:
Good to know!